portfolio/src/main/webapp/index.html

```diff
@@ -25,23 +25,23 @@ <h1 id="introduction">My name is Gábor Fuisz</h1>
           <div class="cardImg" id="faz"></div>
           <div class="cardBottom">
             <p class="cardTitle">Fazekas Mihály Grammar School</p>
-            <span class="cardSecondary">Special Mathematics class</span>
+            <p class="cardSecondary">Special Mathematics class</p>
```

Comment:
Nit: while p and span tags here work quite well, it might be more suitable to e.g. use h2/h3 for the headers.